examples/flannel-conf.yml

```diff
@@ -2,11 +2,11 @@ apiVersion: "kubernetes.cni.cncf.io/v1"
 kind: Network
 metadata:
   name: flannel-conf
-plugin: flannel
-args: '[
-        {
-                "delegate": {
-                        "isDefaultGateway": true
-                }
-        }
-]'
+spec: 
+  config: '{
+    "cniVersion": "0.3.0",
+    type: "flannel",
```

Comment:
It is not a valid json. Nit-pick: Please use the https://jsonlint.com/  for checking the documents

Reply:
Thanks for the eye! I ran it through a linter after a fix, too.